### PR TITLE
feat(docs): Created accessibility-insights-ui package

### PR DESCRIPTION
**Pull request by markreay**
#### Description of changes

We require some components from the web repo to be shared with the docs repo. To facilitate this, this PR creates a package called `accessibility-insights-ui`. This shares a small number of key objects required by the docs repo and also a bundle of css for rendering these items properly.

#### Pull request checklist

- [n/a] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/packages/accessibility-insights-ui/root/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+# accessibility-insights-ui
+
+This library is published was created so the Accessibility Insights website can share the overall
```
Comment (smoralesd) [inline src/packages/accessibility-insights-ui/root/README.md]: I'm not sure if I'm reading this wrong but this part:
```
... is published was...
```
looks odd to me. Is this right?

File: webpack.config.js [hunk 2]
```diff
@@ -180,5 +181,26 @@ const packageReportConfig = {
...
+        pathinfo: false,
+        library: '[name]',
+        libraryTarget: 'umd',
+    },
+    target: 'node',
+};
+
 // For just one config, use "webpack --config-name dev", "webpack --config-name prod", etc
-module.exports = [devConfig, prodConfig, unifiedConfig, packageReportConfig];
+module.exports = [devConfig, prodConfig, unifiedConfig, packageReportConfig, packageUIConfig];
```
Comment (smoralesd) [inline webpack.config.js]: Comment for a different PR: similar to the grunt file... our webpack config keeps growing over time... I guess it's time we ask ourselves: should we split this file into smaller ones?

Comment (markreay) [reply webpack.config.js]: I agree but not sure I want to take this on here.

File: src/packages/accessibility-insights-ui/root/README.md (hunk 1 above)
Comment (markreay) [reply src/packages/accessibility-insights-ui/root/README.md]: I have no idea what I was thinking here. Let me revise. :)

File: Gruntfile.js [hunk 3]
```diff
@@ -576,6 +614,14 @@ module.exports = function(grunt) {
...
         'package-report',
     ]);
+    grunt.registerTask('build-package-ui', [
+        'clean:intermediates',
+        'exec:generate-scss-typings',
+        // 'exec:webpack-prod', // required to get the css assets
+        'exec:webpack-package-ui',
+        'build-assets',
+        'package-ui',
+    ]);
```
[the remark this answers is not in the document]
Comment (markreay) [reply Gruntfile.js]: Oh yes. We should break this up. I agree it's for another PR.

File: src/packages/accessibility-insights-ui/root/README.md (hunk 1 above)
Comment (markreay) [reply src/packages/accessibility-insights-ui/root/README.md]: Fixed the typos.

File: Gruntfile.js [hunk 4]
```diff
@@ -576,6 +614,14 @@ module.exports = function(grunt) {
         'build-assets',
         'package-report',
     ]);
+    grunt.registerTask('build-package-ui', [
+        'clean:intermediates',
+        'exec:generate-scss-typings',
+        // 'exec:webpack-prod', // required to get the css assets
```
Comment (smoralesd) [inline Gruntfile.js]: should we remove this commented line here?

File: pipeline/build-package-ui.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+jobs:
+    - job: 'build_and_publish_package'
+
+      pool:
+          vmImage: 'macOS-10.14'
+
+      steps:
+          - template: install-node-prerequisites.yaml
+
+          - script: yarn lint:check
```
Comment (smoralesd) [inline pipeline/build-package-ui.yaml]: this block including lint, format and copyright checks is repeated on other yaml files too. Does it make sense to extract this to it's own yaml file and the use the `template` directive here and on the other yaml files that use this steps too?

Comment (markreay) [reply pipeline/build-package-ui.yaml]: Yes, probably. I will do it at least for the two packages yamls.

File: Gruntfile.js (hunk 4 above)
Comment (markreay) [reply Gruntfile.js]: I guess so. :)

File: pipeline/build-package-ui.yaml (hunk 5 above)
Comment (markreay) [reply pipeline/build-package-ui.yaml]: done!

File: Gruntfile.js (hunk 4 above)
Comment (markreay) [reply Gruntfile.js]: Removed

**Pull request merged**